bundles/binding/org.openhab.binding.zwave/database/leviton/vrmx1-1lz.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>VRMX1-1LZ</Model>
+	<Label lang="en">Scene Capable Push On/Off Dimmer</Label>
+	<CommandClasses>
+		<Class><id>0x20</id></Class>
+		<Class><id>0x85</id></Class>
+		<Class><id>0x26</id></Class>
+		<Class><id>0x27</id></Class>
+		<Class><id>0x2b</id></Class>
+		<Class><id>0x2c</id></Class>
+		<Class><id>0x91</id></Class>
+		<Class><id>0x72</id></Class>
+		<Class><id>0x73</id></Class>
+		<Class><id>0x77</id></Class>
+		<Class><id>0x86</id></Class>
+	</CommandClasses>
+	
+	<Configuration>
+
+	</Configuration>
+        <Associations>
+                <Group>
```

Comment:
Are any of these association groups used to provide status information back to the controller? If so, you should add the SetToController tag.